x/clp/types/keys.go

```diff
@@ -46,3 +46,20 @@ func GetLiquidityProviderKey(externalTicker string, lp string) []byte {
 	key := []byte(fmt.Sprintf("%s_%s", externalTicker, lp))
 	return append(LiquidityProviderPrefix, key...)
 }
+
+func GetNormalizationMap() map[string]int64 {
```

Comment:
This should be a keeper that only an admin key can change. Otherwise, we have to upgrade the network and manually change this list before we can add new tokens to the dex whose token decimal length is not 10e18.

Reply:
@Jedi2002 Could you please take a look into moving it into keeper?

Reply:
This is not a finalised PR yet , this would be part of the keeper eventually , and we would need a separate TX to add / remove from this list. 
We also need to discuss who gets to execute the add / remove tx .

Reply:
I do propose to limit it to clp - admins as of now , which seems to be the most viable approach

Reply:
Also tbh I feel I, we need to gdo that as a separate pr , this pr can just be for the mathmatical changes and test . Would be easier to review .